benchmark/benchmain/main.go

```diff
@@ -111,6 +111,7 @@ var (
 	serverReadBufferSize  = flags.IntSlice("serverReadBufferSize", []int{-1}, "Configures the server read buffer size in bytes. If negative, use the default - may be a a comma-separated list")
 	serverWriteBufferSize = flags.IntSlice("serverWriteBufferSize", []int{-1}, "Configures the server write buffer size in bytes. If negative, use the default - may be a a comma-separated list")
 	sleepBetweenRPCs      = flags.DurationSlice("sleepBetweenRPCs", []time.Duration{0}, "Configures the maximum amount of time the client should sleep between consecutive RPCs - may be a a comma-separated list")
+	shareConnection       = flag.Bool("shareConnection", true, "Controls whether a single connection or connection per `maxConcurrentCalls` should be used.")
```

Comment:
Instead of this, how about a flag for the number of connections, where each connection does `maxConcurrentCalls` at a time?  Hopefully that wouldn't be too different implementation-wise, and would allow for some more interesting testing options.

Reply:
Sure, done.